tests/integration/ethernet_mtu_test.py

```diff
@@ -80,40 +72,33 @@ def test_decrease_to_zero_iface_mtu():
     desired_state = statelib.show_only(('eth1',))
     origin_desired_state = copy.deepcopy(desired_state)
     eth1_desired_state = desired_state[INTERFACES][0]
-    eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = 0
-    eth1_desired_state['ipv6']['enabled'] = True
 
     with pytest.raises(netapplier.DesiredStateIsNotCurrentError) as err:
         netapplier.apply(copy.deepcopy(desired_state))
     assert '-mtu: 0' in err.value.args[0]
+    # FIXME: Drop the sleep when the waiting logic is implemented.
+    time.sleep(2)
     assertlib.assert_state(origin_desired_state)
 
 
 def test_decrease_to_negative_iface_mtu():
     desired_state = statelib.show_only(('eth1',))
     origin_desired_state = copy.deepcopy(desired_state)
     eth1_desired_state = desired_state[INTERFACES][0]
-    eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = -1
-    eth1_desired_state['ipv6']['enabled'] = True
 
     with pytest.raises(js.ValidationError) as err:
         netapplier.apply(copy.deepcopy(desired_state))
     assert '-1' in err.value.args[0]
     assertlib.assert_state(origin_desired_state)
 
 
-def test_decrease_to_min_ethernet_frame_size_iface_mtu():
+def test_decrease_to_ipv6_min_ethernet_frame_size_iface_mtu():
     desired_state = statelib.show_only(('eth1',))
-    origin_desired_state = copy.deepcopy(desired_state)
     eth1_desired_state = desired_state[INTERFACES][0]
-    eth1_desired_state['state'] = 'up'
-    # the min is 64 - 18 = 46
-    eth1_desired_state['mtu'] = 40
-    eth1_desired_state['ipv6']['enabled'] = True
+    eth1_desired_state['mtu'] = 1280
```

Comment:
Why no test for values below 1280? 1279 would also be a good candidate.

Reply:
syre